.github/workflows/main.yml

```diff
@@ -0,0 +1,88 @@
+name: main
+
+on: [push, pull_request]
+
+jobs:
+  no-naked-pointers:
```

Comment:
By symmetry with "full-flambda", maybe this one should be named "light-no-naked-pointers"?

Reply:
I'm not sure - the test is still quite explicitly one thing, given that it's sole purpose is to allow the debug runtime to trigger warnings in the testsuite? It still builds all the tools and so forth.